scripts/check-copyright

```diff
@@ -0,0 +1,11 @@
+files=`find . -path './tpls' -prune -o -name '*.cpp' -o -name '*.pc.in' -o -name '*.hpp' -o -name '*.cc'`
+echo "" &> scripts/diff_files
+for file in $files; do
+  head -n +15 $file &> header
```

Comment:
probably should use `wc -l LICENSE_FILE_HEADER` instead of a magic number

Reply:
yeah I guess we could do that.